### DISCUSSION TITLE
which vcpkg to use, independent vcpkg.exe or the one comes together with Visual C++?

### DISCUSSION THREAD

**Discussion by felixf4xu** [microsoft/vcpkg#38353] [Q&A]
Hi,

I'm following https://learn.microsoft.com/en-us/vcpkg/get_started/get-started?pivots=shell-cmd#1---set-up-vcpkg to install vcpkg. It installed ok.

But after a while, I realized that I already had another version at `C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\vcpkg\vcpkg.exe`, which can be available if I open a VC development command prompt.

So, in my case, which version of vcpkg.exe should I use?

**Comment by Osyotr** [comment]
Separate copy of vcpkg gives you more flexibility and allows you to pin scripts version.
